suitcase/tests/test_hdf5.py

```diff
@@ -59,87 +62,102 @@ def shallow_header_verify(hdf_path, header, db, fields=None,
                 assert all(np.diff(timestamps) > 0)
 
 
-def test_hdf5_export_single(db_all):
+@pytest.mark.skipif(sys.version_info < (3,5),
+                    reason="bluesky related tests need python 3.5, 3.6")
+def test_hdf5_export_single(db_all, RE):
     """
     Test the hdf5 export with a single header and
     verify the output is correct
     """
-    mds = db_all.mds
-    temperature_ramp.run(mds)
+    RE.subscribe(db_all.insert)
+    RE(count([det], 5, delay = 1), owner="Tom")
     hdr = db_all[-1]
     fname = tempfile.NamedTemporaryFile()
     hdf5.export(hdr, fname.name, db=db_all)
     shallow_header_verify(fname.name, hdr, db=db_all)
 
 
-def test_hdf5_export_single_no_uid(db_all):
+@pytest.mark.skipif(sys.version_info < (3,5),
+                    reason="bluesky related tests need python 3.5, 3.6")
+def test_hdf5_export_single_no_uid(db_all, RE):
     """
     Test the hdf5 export with a single header and
     verify the output is correct. No uid is used.
     """
-    mds = db_all.mds
-    temperature_ramp.run(mds)
+    RE.subscribe(db_all.insert)
+    RE(count([det], 5, delay = 1), owner="Tom")
     hdr = db_all[-1]
     fname = tempfile.NamedTemporaryFile()
     hdf5.export(hdr, fname.name, use_uid=False, db=db_all)
     shallow_header_verify(fname.name, hdr, db_all, use_uid=False)
 
 
-def test_hdf5_export_single_stream_name(db_all):
+@pytest.mark.skipif(sys.version_info < (3,5),
+                    reason="bluesky related tests need python 3.5, 3.6")
+def test_hdf5_export_single_stream_name(db_all, RE):
     """
     Test the hdf5 export with a single header and
     verify the output is correct. No uid is used.
     """
-    mds = db_all.mds
-    temperature_ramp.run(mds)
+    RE.subscribe(db_all.insert)
+    RE(count([det], 5, delay = 1), owner="Tom")
```

Comment:
This is not a like-for-like change.  This should be a `scan` right? 

Same with below.

Reply:
I tried scan with motor delays. But timestamps stay the same. Wait for @tacaswell to fix bugs in ophyd, then try again.

Reply:
xrif: https://github.com/NSLS-II/ophyd/pull/498

Reply:
That fix to ophyd has been merged.